### ISSUE TITLE
@xeokit/xeokit-sdk does not provide an export named 'StoreyViewsPlugin'

### ISSUE THREAD

**Issue by ctrlmaniac** [xeokit/xeokit-sdk#1134]
I'm trying to use the StoreyViewsPlugin with vite but it seems that I am unable to use it because it is not exported from the sdk

The error I get is:

**Unexpected Application Error!**
_The requested module '/node_modules/.vite/deps/@xeokit_xeokit-sdk.js?v=996c991f' does not provide an export named 'StoreyViewsPlugin'_
SyntaxError: The requested module '/node_modules/.vite/deps/@xeokit_xeokit-sdk.js?v=996c991f' does not provide an export named 'StoreyViewsPlugin'

I'm importing it in this way: 

```
import {
  CameraMemento,
  Viewer,
  XKTLoaderPlugin,
  StoreyViewsPlugin,
} from "@xeokit/xeokit-sdk";
```

If I try to include it like it is mentioned in the documentation it gives me an error that says that it doesn't even exists

`import {StoreyViewsPlugin} from '@xeokit/xeokit-sdk/src/plugins/StoreyViewsPlugin/StoreyViewsPlugin.js'`

How do I have to include it? How to fix this issue?

**Comment by xeolabs** [comment]
We have decided to remove StoreyViewsPlugin from the upcoming release because it has not worked well for a while now, and does not work reliably enough as a means of generating storey plans to navigate.

In the meantime the code is still in the repository, but as you know, is not exported in the build: https://github.com/xeokit/xeokit-sdk/tree/master/src/plugins/StoreyViewsPlugin

It still exists in the documentation by mistake - will be removed from the documentation.

If you are able to make this plugin work reliably however, we would consider a PR, and would keep it in the build and the documentation.

The issue that we have with StoreyViewsPlugin are:

- the general technique of making a storey view by taking a canvas snapshot is not particularly robust, because it relies on elements having an appearance (colors, positions etc) that makes a comprehensive plan view image (often the building storeys make a confusing plan view image)
- the storey views snapshots are sized to the extents of the storeys, which can be elongated and make for inconveniently-shaped storey plan images.

In short, it's a fragile way to create storey plan views, which none of our partners are actually using. But if you are able to make it works robustly enough for our sample models, we'd consider a PR

**Comment by ctrlmaniac** [comment]
Yes I am able to make it work it like I wish I do, so please, if you can consider to add it back it would be nice! thank you!

**Comment by xeolabs** [comment]
BTW, until we reintegrate (later today probably) for now you should be able to find it in the v2.3 release and earlier.

**Comment by ctrlmaniac** [comment]
thank you! Can you please tag the release in this issue when it will be available? thank you!

**Comment by xeolabs** [comment]
Hi - StoreyViewsPlugin now restored in @xeokit/xeokit-sdk 2.4.0-alpha-78
